com/win32com/src/extensions/PyFUNCDESC.cpp

```diff
@@ -30,7 +30,7 @@ PyObject *PyObject_FromFUNCDESC(FUNCDESC *desc) { return new PyFUNCDESC(desc); }
 BOOL PyObject_AsFUNCDESC(PyObject *ob, FUNCDESC **ppfd)
 {
     BOOL rc = FALSE;
-    if (ob->ob_type != &PyFUNCDESC::Type) {
```

Comment:
It would be great if these kinds of changes were split into their own PR - they are probably preferred regardless of whether we can get this no-gil branch working.

Reply:
Missed some: I only switched out `ob_type` for files that caused build failures for 3.14 on CI